### PR TITLE
WIP: fix error in principle axis

### PR DESCRIPTION
**Pull request by kain88-de**
Changes made in this Pull Request:
 - fix principle_axis calculation

This is strangely covered in VALID tests even though the code should have failed due to missing imports. I'll have to look closer into the tests

PR Checklist
------------
 - [x] Tests?
 - [ ] ~~Docs?~~
 - [ ] ~~CHANGELOG updated?~~
 - [ ] ~~Issue raised/referenced?~~

### CODE REVIEW

File: package/MDAnalysis/core/topologyattrs.py [hunk 1]
```diff
@@ -748,17 +747,13 @@ def principal_axes(group, **kwargs):
         .. versionchanged:: 0.8 Added *pbc* keyword
         """
         atomgroup = group.atoms
-        pbc = kwargs.pop('pbc', MDAnalysis.core.flags['use_pbc'])
-
-        if pbc:
-            eigenval, eigenvec = eig(atomgroup.moment_of_inertia(pbc=True))
-        else:
-            eigenval, eigenvec = eig(atomgroup.moment_of_inertia(pbc=False))
+        pbc = kwargs.pop('pbc', flags['use_pbc'])
```
Comment (orbeckst) [inline package/MDAnalysis/core/topologyattrs.py]: Are you still importing `flags` somewhere?

Comment (kain88-de) [inline package/MDAnalysis/core/topologyattrs.py]: Top of the file

File: package/MDAnalysis/core/topologyattrs.py [hunk 2]
```diff
@@ -30,7 +30,6 @@
 from ..exceptions import NoDataError, SelectionError
 from .topologyobjects import TopologyGroup
 from . import selection
-from . import flags
```
Comment (kain88-de) [inline package/MDAnalysis/core/topologyattrs.py]: This was a double import. You can see the first import of flags if you look at the whole file.

**Comment by kain88-de** [comment]
Tests are activated again. I also now removed the flag check. The comments in the test_atomgroup file say that aren't supposed to be used anymore.

**Comment by kain88-de** [comment]
ping @richardjgowers 

the principle axis tests are working now. agurment parsing is better now and flags handling is documented for the function.

**Pull request merged**